### ISSUE TITLE
Add issues and GitHub link to the live doc

### ISSUE THREAD

**Issue by jaygambetta** [Qiskit/openqasm#69]
I would like to have at the top links to making pull requests, issues, and a link to the GitHub. This is important as when you are the live documentation it is not clear how to edit it or make comments. I will make a short term change to the introduction for this but long term i think a header would be good.

**Comment by RasmitDevkota** [comment]
Is this still an active issue of interest?

**Comment by RasmitDevkota** [comment]
Yes, I am! I have recently began navigating the repository, which area of the live doc would we want to include this in?

**Comment by mbhealy** [comment]
We've already got some static links in the navigation pane on the sidebar. Perhaps adding a link to the github there would be sufficient.

**Comment by Taaahu** [comment]
Hey I want to claim this issue @jaygambetta

**Comment by mbhealy** [comment]
@levbishop do you have any thoughts on whether this issue should still be open?
